### DISCUSSION TITLE
Workaround for missing "expand sidebar on hover"

### DISCUSSION THREAD

**Discussion by matdave** [zen-browser/desktop#3494] [General]
The latest update seems to remove the option to expand the sidebar on hover. I found a decent workaround is to enable compact mode -> leave toolbar visible -> set the sidebar to expanded. It works similarly to how it was before, except you lose the ever present sidebar. Hopefully the expand sidebar on hover comes back, but this works for now.

**Comment by scoobydooxp** [comment]
Check the Zen mod - Sidebar Expand on Hover

It's not perfect but it is a start.

**Reply by matdave** [reply]
This works well enough (aside from missing close icons!)

**Comment by Frederic-C** [comment]
The keyboard shortcut is also missing, I was using it a lot.

**Reply by V-in** [reply]
Me too. Also by default the top buttons ( settings, toggle width itself ) move from top bar to side bar depending on whether or not it's expanded, which does not look natural when toggling width - I think because previously the semantics where basically "toggle to see the full tab names", now it's more like an entire layout shift and that's not what I expect, I just wanted to check out the full tabs! For layout I'd just switch compact mode on and off

I honestly just downgraded to 1.0.1-a.22 for now until I can get around to customizing the newest version to match the old behavior

**Comment by mateuszkozakiewicz** [comment]
This was one of the core features of the browser, even the Firefox team is working on implementing it into the stock browser because of the demand and Zen had it and removed it? How does that make sense?

**Reply by Gorgaer** [reply]
This feature is literally the reason why I switched to Zen from Firefox without much thought. Such functionality can be obtained with addons, but in Zen it worked right away and looked beautiful.
